readthedocs/restapi/views/footer_views.py

```diff
@@ -43,7 +43,7 @@ def get_version_compare_data(project, base_version=None):
     }
     if highest_version_obj:
         ret_val['url'] = highest_version_obj.get_absolute_url()
-        ret_val['slug'] = highest_version_obj.slug,
+        ret_val['slug'] = highest_version_obj.slug
```

Comment:
Removing this comma produces tests to fail... I had the same problem and I ended up writing

    (highest_version_obj.slug, )

Reply:
I think the test should be fixed. Its expecting a tuple but it should get the slug in sting

Reply:
@safwanrahman agreed -- want to PR it to master, and I'll merge it in here?